packages/db/prisma/schema.prisma

```diff
@@ -843,60 +904,209 @@ enum ModerationType {
   OPENAI_FLAGGED
   OPENAI_OVER_THRESHOLD
   LLM_REFUSAL
+
+  @@schema("public")
 }
 
 enum TranscriptVariant {
   ORIGINAL
   COMPRESSED
+
+  @@schema("public")
 }
 
 enum TranscriptStatus {
   PENDING
   FAILED
   SUCCESS
+
+  @@schema("public")
 }
 
 enum SnippetStatus {
   PENDING
   FAILED
   SUCCESS
+
+  @@schema("public")
 }
 
 enum LessonSummaryStatus {
   PENDING
   GENERATED
   FAILED
   SUCCESS
+
+  @@schema("public")
 }
 
 enum LessonPlanStatus {
   PENDING
   GENERATED
   FAILED
   SUCCESS
+
+  @@schema("public")
 }
 
 enum LessonPlanPartStatus {
   PENDING
   GENERATED
   FAILED
   SUCCESS
+
+  @@schema("public")
 }
 
 enum SnippetCompression {
   BREVITY
   SIMPLIFICATION
   RELEVANCE
+
+  @@schema("public")
 }
 
 enum QuizQuestionStatus {
   PENDING
   FAILED
   SUCCESS
+
+  @@schema("public")
 }
 
 enum QuizAnswerStatus {
   PENDING
   FAILED
   SUCCESS
+
+  @@schema("public")
+}
+
+model Ingest {
+  id          String    @id @default(cuid())
+  status      String
+  createdAt   DateTime  @default(now()) @map("created_at")
+  updatedAt   DateTime  @updatedAt @map("updated_at")
+  completedAt DateTime? @map("completed_at")
+
+  ingestLessons           IngestLesson[]
+  ingestLessonTranscripts IngestLessonCaptions[]
+  ingestLessonPlans       IngestLessonPlan[]
+  ingestLessonPlanParts   IngestLessonPlanPart[]
+  ingestErrors            IngestError[]
+  ingestOpenAiBatches     IngestOpenAiBatch[]
+
+  @@map("ingest")
+  @@schema("ingest")
+}
+
+model IngestLesson {
+  id           String   @id @default(cuid())
+  oakLessonId  Int      @map("oak_lesson_id")
+  ingestId     String   @map("ingest_id")
+  data         Json     @map("data") @db.JsonB
+  dataHash     String   @map("data_hash")
+  step         String
+  stepStatus   String   @map("step_status")
```

Comment:
Currently got `step` and `stepStatus` as text (i.e. not enums) with the typesafety coming from ts helpers, but I'm thinking of using enums instead.